packages/translate-results/lib/texts.ts

```diff
@@ -0,0 +1,54 @@
+export const texts = Object.freeze({
+	'startupError.blasterError': () => 'Cannot continue: IR blaster missing or incorrectly attached',
+	'startupError.blasterError.desc': () => 'Infrared blaster assigned to the device is missing or malfunctioning. Check the wiring, replace the blaster or assign another working CandyBox port to this device',
+	'startupError.bootstrappedPlatformError': () => 'Cannot continue: Suitest bootstrap app is not running',
+	'startupError.bootstrappedPlatformError.desc': () => 'Suitest tried to start the bootstrap application on this device but failed several times and will try no more. Please connect to the device and start the bootstrap app manually, then disconnect and the scheduled test will continue. If you have configured the Suitest channel, tune the TV to this channel and verify that Suitest badge is displayed on TV in the top right corner. If you have not configured the Suitest channel, please contact support',
+	'startupError.testQueued': () => 'Test is added to queue',
+	'startupError.testQueued.desc': () => 'Execution will start as soon as other tests queued on this device will finish execution',
+	'startupError.noAvailableAutomatedMinutes': () => 'Cannot continue: you\'ve used up all of your testing minutes',
+	'startupError.noAvailableAutomatedMinutes.desc': () => 'You testing a lot! How about [getting a bigger subscription](https://the.suite.st/preferences/billing)? Or, if you would like to purchase more testing minutes for the current billing cycle, please contact [sales@suite.st](mailto:sales@suite.st). Your testing minutes will renew',
```

Comment:
will `[sales@suite.st](mailto:sales@suite.st)` work in console (js api)?

Reply:
Think that it will be handled on jsapi side